### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aiamuzz**
self descriptive ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,19 +9,31 @@ This way your Wireless debugging(WiFi ADB) will be secure and private(SAFE)
...
+- Toggle WiFiADB - ON/OFF using Magisk Manager App.
 
 ## Usage
 
-Enable this module in magisk.
+**Configuring WiFiADB to start on boot :**
+
+- Download the latest release of the module.
+- Install it manually using the Magisk Manager App
+- Reboot Device.
```
Comment (mrh929) [inline README.md]: After installing this module, you can enable it immediately (switch the button on) without reboot, and service will start in next boot.

File: README.md [hunk 2]
```diff
@@ -44,3 +56,18 @@ setprop service.adb.tcp.port 5555
...
 start adbd
 ```
+
+## NOOB steps to configure your private port and enabling log
+
+```
+adb shell
+su
+cd /data/adb/modules/magisk-wifiadb/
+echo -e "ADB_PORT=1234\nENABLE_LOG=1" >> config
```
Comment (mrh929) [inline README.md]: I suggest:
echo -e "ADB_PORT=1234\nENABLE_LOG=1" > config

File: README.md [hunk 3]
```diff
@@ -9,19 +9,31 @@ This way your Wireless debugging(WiFi ADB) will be secure and private(SAFE)
...
+
+**Toggling WiFiADB ON/OFF :**
+Open Magisk Manager App -> Modules
+
+- turning the module switch to OFF will disable WiFiADB  (Device reboot not required)
+- turning the module switch to ON will enable WiFiADB  (Device reboot not required)
 
-### Configuration
+Note : If you accidentally happen to reboot device when this module switch is turned OFF then WiFiADB will not be started on boot.
+STEPS
```
Comment (mrh929) [inline README.md]: consider removing it?

Comment (aiamuzz) [reply README.md]: reworded ...

**Pull request merged**